package-lock.json

```diff
@@ -1,8 +1,3448 @@
 {
 	"name": "vscode-debug-adapter-apache-camel",
 	"version": "0.5.0",
-	"lockfileVersion": 1,
+	"lockfileVersion": 2,
```

Comment:
Done automatically by npm audit fix. It would be better to do it in a separate repo but the change seems small enough to do both at the same time. What do you think Aurelien?

Reply:
the change is 3400+ lines of modified lines. That's not small. better to do in a separate Pull request